### ISSUE TITLE
Gmail reports plugin unsafe

### ISSUE THREAD

**Issue by myshoeshurt** [guh/guh#397]
After adding gmail as a device, the ever-so-vigilant gmail team sent me an e-mail: 

		
Hi George,
Google just blocked someone from signing into your Google Account g***@gmail.com from an app that may put your account at risk.

Are you the one who tried signing in?
Google will continue to block sign-in attempts from the app you're using because it has known security problems or is out of date. You can continue to use this app by allowing access to less secure apps, but this may leave your account vulnerable.

I recognise that this might not be an important problem :)

**Comment by t-mon** [comment]
Strange, I cannot reproduce this behaviour. Does it work if you tell in your Gmail account that it was you? I know that behaviour when I'm abroad. All my Gmail accounts are blocked if I'm logging in from a different country until I confirm that I am who I am.

**Comment by myshoeshurt** [comment]
No, the issue would happen if I change my current *default* security
settings on gmail. The standard security settings of a fresh, newly created
gmail account will report the problem.

**Comment by t-mon** [comment]
Note: currently this plugin works only with weak gmail security settings. Need to be reworked and enable all security mechanisms.
